Automattic/Gravatar-SDK-iOS
# Simpilfy strings generation logic

Closes #

### Description

This updates the `wpmreleasetoolkit` to `12.2`, since `12.1` introduced support for specifying the encoding of the `Localizable.strings` files.  Now we rely on the `ios_generate_strings_file_from_code` lane to handle the UTF-8 encoding, rather than handling that logic ourselves.

### Testing Steps

1. Add a new `SDKLocalizedString()` to a source file in the`GravatarUI` module
2. Run `bundle exec fastlane generate_strings skip_commit:true`
- [ ] OBSERVE that the `Localizable.strings` file is `UTF-8` encoded.  You can check the encoding with the command below.

```shell
$ file -I Sources/GravatarUI/Resources/en.lproj/Localizable.strings
Sources/GravatarUI/Resources/en.lproj/Localizable.strings: text/plain; charset=utf-8
```
